### DISCUSSION TITLE
Add jar file in release for easier trial

### DISCUSSION THREAD

**Discussion by hieuwu** [LemonAppDev/konsist#1378] [Request a Feature] [answered]
### Tell us more.

Hi guys currently I want to demonstrate konsist in my workplace, however due to some scurity check I can not pull and build the code of konsist or pull the library directly from mavenCentral().
It would be greate if we can have a jar file in the release so that it would be easier to try the library directly

**Comment by igorwojda** [comment, marked as answer]
Every artifact cna be downloaded directly from MavenCentral.
https://s01.oss.sonatype.org/content/repositories/releases/com/lemonappdev/konsist/

**Reply by hieuwu** [reply]
Thank you
